src/libawkward/forth/ForthMachine.cpp

```diff
@@ -1358,7 +1358,7 @@ namespace awkward {
       try {
         value = (int64_t)std::stoul(word.substr(2, word.size() - 2), nullptr, 16);
       }
-      catch (std::invalid_argument err) {
+      catch (std::invalid_argument& err) {
         return false;
       }
```

Comment:
I haven't been consistently catching references to exception objects, rather than copies. But there isn't a lot of `try`-`catch` logic in the codebase—this isn't something that needs to be made consistent just yet.